src/features/page_view_event/aggregate/index.js

```diff
@@ -135,10 +135,10 @@ export class Aggregate extends FeatureBase {
     // Capture query bytes sent to RUM call endpoint (currently `1`) as a supportability metric. See metrics aggregator (on unload).
     agentRuntime.queryBytesSent[protocol] = (agentRuntime.queryBytesSent[protocol] || 0) + queryString?.length || 0
 
-    const isValidJsonp = submitData.jsonp(
-      this.getScheme() + '://' + info.beacon + '/' + protocol + '/' + info.licenseKey + queryString,
+    const isValidJsonp = submitData.jsonp({
+      url: this.getScheme() + '://' + info.beacon + '/' + protocol + '/' + info.licenseKey + '?' + queryString,
```

Comment:
Just double checking the `queryString` does not already have the `?` character.

Reply:
correct